bats/scripts/package.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
It would be good for the script to have a comment block describing what it does. `bats-lint.pl` in the same directory does.

Reply:
> It would be good for the script to have a comment block describing what it does.

I've moved the script logic into the `Makefile`, to take advantage of dependency tracking (not downloading/building when the target already exists). I've added a comment, but let me know if you feel it is not sufficient!